protocol/messenger_backup_handler.go

```diff
@@ -172,6 +184,9 @@ func (m *Messenger) handleBackedUpProfile(message *protobuf.BackedUpProfile, bac
 			}
 			err = m.multiAccounts.StoreIdentityImages(message.KeyUid, idImages, false)
 			response.AddImages(idImages, err == nil)
```

Comment:
I would move this line after error checking, no need to add to response if an error is thrown, better to just fail immediately, similar above we should do the same